src/spec.md

```diff
@@ -130,6 +130,46 @@ webxdc.sendToChat({
 
 > If you want to store text don't use `btoa()`, rather use `message.file.plainText` directly, because [`btoa()` has problems with some unicode/emojis](https://developer.mozilla.org/en-US/docs/Web/API/btoa#unicode_strings)
 
+### importFiles()
+
+```js
+let files = await window.webxdc.importFiles(filters);
```

Comment:
```suggestion
let files = await window.webxdc.importFiles(options);
```

i think, "filters" is not really matching here, also in its plural form (it is maybe inspired by https://www.electronjs.org/docs/latest/api/dialog - but there there user can really select one filter from several defined ones - but this is not what we want here)

another name would be "accept". otoh, it is not super-important as it is not really part of the api the user "types" at the end, so no blocker